.github/workflows/codeql-analysis.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         language:
           - c-cpp
           - python
+          - actions
```

Comment:
This is from the commit of #4940. We should either merge #4940 (and see the new security issues), then update this PR, or, remove it from this PR (thus losing the way to verify that the issue isn't there anymore).

Reply:
This PR has been rebased since. Resolving...